### PR TITLE
WD : Feat category and video listing

### PR DESCRIPTION
**Pull request by devananth**
## Complete Category and Video Listing

  - Created *category context* for listing categories
  - Created *video context* for listing videos
  - Added **sortBy** by date functionality too.

### Files to be checked:

**Contexts:**
      - category-context
      - video-context

**Pages:**
      - Explore Page

**Components:**
      - Chips
      - VideoCard

**custom hooks:**
      - useAxios

**utils**
      - getSelectedCategoryVideos
      - getSortedVideos

**Demo Gif:**

LensPlay_videoListing

### CODE REVIEW

**Review by spraveenofficial — APPROVED** [review]

**Review by mnath30 — APPROVED** [review]
Looks good to merge

File: src/App.css [hunk 1]
```diff
@@ -32,6 +32,75 @@ body{
...
+    margin-bottom: 3rem;
+}
+  
+
+  
+.sort__dropdown {
+    position: absolute;
+    top: 90%;
+    left : 20%;
+    z-index: 999;
```
Comment (rahul31agrawal) [inline src/App.css:79]: Z-index should be between 1 to 10

File: src/App.css [hunk 2]
```diff
@@ -32,6 +32,75 @@ body{
...
+    background-color: var(--light-theme-bg-color);
+    width : max-content;
+    border : 1px solid var(--primary-color);
+   
+}
+
+.sort__dropdown li{
+    font-size: 1rem;
+    border-bottom : 1px solid var(--primary-color);
+    padding : 3px 7px;
```
Comment (rahul31agrawal) [inline src/App.css:89]: use rems instead of px

**Review by rahul31agrawal — APPROVED** [review]
Hi Devanath,

1. The UI looks awesome
2. The code structure is neat
3. optional sugestion: remove borders from video cards, it will look good
4. optional sugestion: Increase the size of the video card
Thanks

File: src/App.css (hunk 2 above)
Comment (devananth) [reply src/App.css:89]: It's below 10px so I have used px.

**Pull request merged**